flask_restplus/marshalling.py

```diff
@@ -44,14 +50,22 @@ def make(cls):
         fields = apply_mask(fields, mask, skip=True)
 
     if isinstance(data, (list, tuple)):
-        out = [marshal(d, fields) for d in data]
+        out = [marshal(d, fields, skip_none=skip_none) for d in data]
         if envelope:
             out = OrderedDict([(envelope, out)])
         return out
 
-    items = ((k, marshal(data, v) if isinstance(v, dict)
-              else make(v).output(k, data))
-             for k, v in fields.items())
```

Comment:
The generator syntax is important for performance.
You can nest it with another generator (and factorize the `if skip_none`:
```python
if skip_none:
    items = ((k, v) for k, v in items if v is not None)
```
So it's still a generator and the `skip_none` condition is evaluated once